ArcGISRuntimeSDKQt_QMLSamples/Layers/LoadWfsXmlQuery/LoadWfsXmlQuery.qml

```diff
@@ -0,0 +1,71 @@
+// [WriteFile Name=LoadWfsXmlQuery, Category=Layers]
+// [Legal]
+// Copyright 2019 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.6
+import Esri.ArcGISRuntime 100.6
+
+Rectangle {
+    id: rootRectangle
+    clip: true
+    width: 800
+    height: 600
+
+    readonly property string xmlQueryString :
+     '
```

Comment:
Genuinely surprised single ticks work here for multiline strings! Backticks (`) tend to be the accepted standard for multiline strings in Javascript. Not sure if case for QML too?

Reply:
@anmacdonald yep it works. The JS backtick is new ES6 functionality and the default Qt Creator doesn't yet support it, so I opted to use this instead since it works well. Otherwise it will run but creator will complain about syntax errors OOTB